pkg/controller/sync/resource.go

```diff
@@ -244,17 +242,13 @@ func (r *federatedResource) RecordEvent(reason, messageFmt string, args ...inter
 	r.eventRecorder.Eventf(r.Object(), corev1.EventTypeNormal, reason, messageFmt, args...)
 }
 
-func (r *federatedResource) NewUpdater() FederatedUpdater {
-	return NewFederatedUpdater(r.informer, r)
-}
-
 func (r *federatedResource) overridesForCluster(clusterName string) (util.ClusterOverridesMap, error) {
 	r.Lock()
 	defer r.Unlock()
 	if r.overridesMap == nil {
 		overridesMap, err := util.GetOverrides(r.federatedResource)
 		if err != nil {
-			return nil, errors.Errorf("Error reading cluster overrides for %s %q", r.federatedKind, r.federatedName)
+			return nil, errors.Wrapf(err, "Error reading cluster overrides")
```

Comment:
This change seems not relevant to this PR. Is this because the Updater (now dispatcher) deciphers and appends this information from the resource?

Reply:
This was a latent bug discovered in the process of implementing this PR.  Since this error is only ever logged as an event on the resource, there is zero value in including the kind and name.

Reply:
Ack.